views/HakaiWaterPropertiesInstrumentProfileResearch.sql

```diff
@@ -7,14 +7,13 @@ FROM
     ctd.ctd_post_qc_data d
 WHERE
     (
-        d.cast_processing_stage >= '8_binAvg' :: ctd.processing_stage
-        OR d.cast_processing_stage >= '8_rbr_processed' :: ctd.processing_stage
+        d.cast_processing_stage >= '10_qc_pi' :: ctd.processing_stage
```

Comment:
the processing_stage seemed wrong to me. but maybe it was ment to be the same for reasearch and provisional, not sure